### ISSUE TITLE
Inconsistant feature visualization settings for different tools

### ISSUE THREAD

**Issue by suricactus** [qgis/QGIS#35129]
There is big inconsistency in the QGIS settings regarding the feature visualization when certain tools are active. I find this ugly and confusing, therefore a bug report instead of feature request. 

### A summary table of the problem:

| setting\tool | Selection | Digitize | Identify | Measure |
|--------------|-----------|----------|----------|---------|
|fill color |*|*|*|*|
|line color ||*|||
|line width ||*|||
|min line width |||*||
|buffer width |||*||

### Visual support of the problem

![Screenshot_20200315_162800](https://user-images.githubusercontent.com/2820439/76824257-40e13f80-681f-11ea-8deb-e91f25a4ad86.png)
![Screenshot_20200315_162807](https://user-images.githubusercontent.com/2820439/76824258-42126c80-681f-11ea-8a23-a1b0ff9252fd.png)
![Screenshot_20200315_162820](https://user-images.githubusercontent.com/2820439/76824415-b0572f00-681f-11ea-93a1-69e942dce267.png)

### Proposed solution
1) The best solution would be that one can select/create a layer style the same way as they do it for a layer style.
2) A simple solution would be able to configure the same settings for each tool, which in my opinion should be:
- fill color
- line color
- line width

Once we agree on a solution for this, I can apply the changes.

**Comment by uclaros** [comment]
Also #23261 could be included to your suggestion

**Comment by Pedro-Murteira** [comment]
Can still replicate on QGIS 3.16.14 and 3.22.1.

**Comment by uclaros** [comment]
Can you clarify what exactly the bug is? From my understanding this is a feature request to allow full customization of selection, identify, rubberband and measure rubberband appearances

**Comment by Pedro-Murteira** [comment]
you are correct, I can replicate what was described on the ticket, meaning the inconsistency still remains, but it should be considered a feature request instead of a bug.
